libraries/DataFlash/LogFile.cpp

```diff
@@ -1268,10 +1268,13 @@ void DataFlash_Class::Log_Write_EKF(AP_AHRS_NavEKF &ahrs, bool optFlowEnabled)
                 HAGL : (int16_t)(100*HAGL),
                 offset : (int16_t)(100*gndOffset),
                 RI : (int16_t)(100*rngInnov),
-                  meaRng : (uint16_t)(100*range),
-                  errHAGL : (uint16_t)(100*gndOffsetErr)
-               };
-              WriteBlock(&pkt5, sizeof(pkt5));
+                meaRng : (uint16_t)(100*range),
+                errHAGL : (uint16_t)(100*gndOffsetErr),
+                angErr : 0.0f,
+                velErr : 0.0f,
+                posErr : 0.0f
```

Comment:
That was precisely my point, isn't the EKF5 only used by the EKF1? EKF2 uses NKF5, which I believe is the only one that needs to be changed.